modules/developer_manual/pages/general/devenv.adoc

```diff
@@ -1,17 +1,12 @@
 = Setup Your Development Environment
+:toc: right
 
-To be able to use and develop with ownCloud follow these steps.
+== Introduction
 
-1.  xref:install-the-core-software[Install the Core Software]
-2.  xref:install-dependencies-on-opensuse-leap-42.3[Setup the web server and database]
-3.  xref:setup-the-webserver-and-database[Get the source]
-4.  xref:set-user-group-and-permissions[Install software dependencies]
-5.  xref:enable-debug-mode[Enable debug mode]
-6.  xref:setup-owncloud[Setup ownCloud]
+This page helps to setup your environment for use and develop with ownCloud.
```

Comment:
This sentence doesn't read quite right. How about: "This page helps with setting up your environment for use with and developing ownCloud."?